### PR TITLE
changed the link from workflow to choosing runner for a job in the FP…

### LINKED ISSUE #15966: [User feedback] Add link to actions info on runner choice
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[Configuring code scanning](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#specifying-an-operating-system)

### What part(s) of the article would you like to see updated?

Question from user:

> what other options are there than ubuntu-latest?

Docs update needed:

We should replace the link from the section "Specifying an operating system" to  "Workflow syntax for GitHub Actions" with a link to [Choosing the runner for a job](https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job). Then it will be easy for users to find out more about runner types.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by cmwilson21**
…T version only

### Why:

Closes [15966](https://github.com/github/docs/issues/15966)

### What's being changed:
Changed the paragraph from reference the GitHub workflow syntax link to the link for Choosing the runner for a job. This change was only made in the FPT version. 

The FPT versioning previously linked to - Workflow syntax for GitHub Actions [jobs.id.runs_on](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on). It now links to [Choosing the runner for a job](https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job).

The GHES versioning links to the GHES version of the same article - [Workflow syntax for GitHub Actions](https://docs.github.com/en/enterprise-server@3.4/actions/using-workflows/workflow-syntax-for-github-actions#self-hosted-runners)

Should the GHES versioning reference [Choosing the runner for a job](https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job) as well?

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@cmwilson21 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by MrWright12 — COMMENTED** [review]
Very well done

File: content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md
Comment (mchammer01) [inline content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md:183]: Not sure whether this is correct, but the conditions covered by the `{% else %}` here are FPT (which we want), GHEC and GHAE. Do we want to link to "Choosing the runner for a job" for GHEC AND GHAE too?

I'll defer to the author of the issue here as I'm not sure what the best article we should be linking to for GHAE.

**Review by mchammer01 — COMMENTED** [review]
@cmwilson21 - thanks for making this improvement ✨ 
Your update looks good but as well as linking to the suggested article for FPT, it also links to that article for both GHEC and GHAE. I am not sure what article(s) we want to link to for both GHEC and GHAE. 
Roping @felicitymay here as the issue author, for further clarification.

I'm not sure that linking to https://docs-16948-fcaa40.preview.ghdocs.com/en/github-ae@latest/actions/using-jobs/choosing-the-runner-for-a-job for GHAE is very helpful 🤔

File: content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md
Comment (felicitymay) [reply content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md:183]: @cmwilson21  and @mchammer01 - great questions. Apologies for not considering the versioning when I raised the issue. As far as I can tell, both the articles [Choosing the runner for a job](/actions/using-jobs/choosing-the-runner-for-a-job) and [Using labels with self-hosted runners](/actions/hosting-your-own-runners/using-labels-with-self-hosted-runners) are now available for all versions of the docs. So it looks as if we could simplify this and remove the conditions entirely, so that we link to both articles for all versions.

I _think_ that when this was first written maybe we didn't support self-hosted runners on GitHub.com or GHAE, but they are clearly supported now.

Comment (cmwilson21) [reply content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md]: @mchammer01 and @felicitymay - thank you both for weighing in on this! I removed the versioning, which in turn did remove the reference for "[Workflow syntax for GitHub Actions](/actions/reference/workflow-syntax-for-github-actions#self-hosted-runners)". Let me know if you still want that included in the links as well.

I also repositioned the "For more information" line in the very next paragraph. It was on it's own line, but I removed the break as I thought it made sense for it to be included with the paragraph it was referring to, the self-hosted runners bit. If that needs to be reverted, let me know and I'm happy to change it back. I'm adding screenshots to show the difference for your reference. 

Thank you both, again!

Screen Shot 2022-04-13 at 1 33 22 PM
Screen Shot 2022-04-13 at 1 39 13 PM

File: content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md
Comment (felicitymay) [inline content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md:183]: Hi @cmwilson21 👋🏻 

I'm guessing that you might have left this here in a comment in case you needed to revert your changes.

```suggestion
```

Your changes look great in the [preview](https://docs-16948-fcaa40.preview.ghdocs.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#specifying-an-operating-system), so I'd recommend deleting this line ✨

**Review by felicitymay — DISMISSED** [review]
👋🏻 Hi Courtney, this looks ready to merge when you've deleted the comment with the old content. It was a good call to combine the two other sentences into a single paragraph 💖

**Review by felicitymay — APPROVED** [review]

File: content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md
Comment (cmwilson21) [reply content/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning.md]: That's exactly what I did. I told myself not to forget to delete it but oh well, lol. Thank you, @felicitymay :D

**Review by Setland34 — COMMENTED** [review]
PR/fellowship

**Review by Setland34 — COMMENTED** [review]
PR/code

**Review by Setland34 — COMMENTED** [review]
PR/GO

**Comment by dejadior440** [comment]
PR/GO

**Comment by dejadior440** [comment]
PR/fellowship

**Pull request merged**
